### ISSUE TITLE
Opportunity: Open Collective contract for Canada

### ISSUE THREAD

**Issue by patcon** [hyphacoop/organizing#460]
Blocked on https://github.com/hyphacoop/organizing/issues/374

<sup>_This initial comment is collaborative and open to modification by all._</sup>
## Summary

**Pace:** slow, potentially year-long through 2021
**Contact:** Pia Mancini, Co-founder, Open Collective
**GDrive:** https://drive.google.com/drive/u/0/folders/1aAExZzaUZG7PzcF1PGVOJqg5xfhXTKIo

#### Timeline
- `2020-01-06` Pia msg'd patcon on OC Slack
  - "are you still in Canada? I'm thinking of opening up a fiscal host there, is this something that could interest you? Otherwise, anyone who I can talk to?"
- `2021-02-04` OC publishes [2020 in Review & 2021 Update](https://blog.opencollective.com/2020-in-review-2021-update/)
  - "we have plans to move more into Europe, the UK, and (potentially) Canada this year"
- `2021-02-06` patcon logs into Slack and restarts discussion
- `2021-02-09` Pia suggests contracting opportunity
- `2021-02-11` patcon brings opp to WGs
- `2021-02-24` patcon<>pia call
- `2021-02-25` bizdev +1 small proposal

#### Status
- logic: OC would prefer a partner run in Canada, but OC:
    1. already have lots of knowledge, and
    2. can mutualize costs easily.
- options
  1. OC spins up org and part-time contracts Hypha member to be Canadian connection for OC Canada
      - person timeshares between OC and Hypha
      - OC would handle admin for OC Canada
      - OC Canada would focus on all Canada, Hypha would focus "on local co-ops" (Pia misunderstands us for now)
  2. something else

## To Do

- OC relationship
  - [x] prelim discussion in bizplanning `2021-02-11`
  - [x] prelim discussion in operations `2021-02-12`
  - [x] book info-gathering call with Pia: FEB 24
    - [x] 2021-02-24 host call
    - [x] archive [notes](https://meetings.hypha.coop/2021-02-24-open-collective.html)
    - [x] report back to bizplanning (should have been operations) FEB 25 APPROVED
  - [ ] draft external proposal for first engagement
- related: OC promo video sponsorship. [summary doc.](https://docs.google.com/document/d/1SxB-B1CTrL41bxW0VlOjk4qumgugou6zjSKyffBoHwY/edit)
  - [x] propose in bizdev: FEB 11 https://github.com/hyphacoop/organizing-private/pull/119
  - [x] summarize proposal. [pad](https://hackmd.io/qtnf6yxeRWyOL9ChdzggkA?view)
  - [x] announce ~~at all hands~~ in chat
  - [x] solicit status from finance. +1 via chat
  - [x] yes/no in bizdev FEB 25 [yes]
  - yes/no to OC ~~FEB 26~~ MAR 5 note: not sure this call happened
    - first planning call
    - [x] share availability -- @patcon
- proposal candidate todos
  - contact ONN to seek contacts, [re: Shared Platform](https://theonn.ca/our-work/our-regulatory-environment/shared-platforms/)
  - contact Brian Iler

**Comment by writerly03** [comment]
Open Collective decided in 2022 to focus on the US. We continue to work with them as a fiscal host, but working on the larger partnership is on hold indefinitely
